v_5_0_0/files/conf/k8s-addons

```diff
@@ -141,6 +141,19 @@ do
     sleep 5s
 done
 
+# apply network policies:
+NETWORK_POLICIES_FILE="network_policies.yaml"
+NAMESPACES="default giantswarm"
+for namespace in ${NAMESPACES}; do
+    while
+      /usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /srv:/srv -v /etc/kubernetes:/etc/kubernetes $KUBECTL apply -f /srv/$NETWORK_POLICIES_FILE -n $namespace
```

Comment:
Do we need to apply this on newly created namespaces as well?

Reply:
I mean, if the user creates a new namespace after the cluster is up.

Reply:
The AC of the tasks says Network Policies for `kube-system `and `giantswarm`. don't think it is necessary for other namespaces. we are not in charge to control other customer namespace here.

oh noted that I've put there `default` rather than `kube-system`